tasks/single-zone.yml

```diff
@@ -1,50 +1,47 @@
 ---
-- name: Configure {{ item.name }} zone
+- name: Configure {{ zone.name }} zone
   command:
     cmd: >
-      /usr/local/bin/pdns-auth-api-upsert-zone "{{ pdns_auth_api_connect }}" "{{ pdns_auth_api_server }}"
-      "{{ item.name }}" "{{ item.kind | default('Master') }}" "{{ item.soaEdit | default('DEFAULT') }}" "{{ item.soaEditApi | default('') }}"
-      "{{ item.dnssec | default(false) }}" "{{ item.nsec3Iterations | default(5) }}" "{{ item.nsec3Salt | default('dada') }}" {{ item.defaultNameservers }}
+      /usr/local/bin/pdns-auth-api-upsert-zone "{{ pdns_auth_api_connect | mandatory }}" "{{ pdns_auth_api_server }}"
+      "{{ zone.name }}" "{{ zone.kind | default('Master') }}" "{{ zone.soaEdit | mandatory }}" "{{ zone.soaEditApi | default('') }}"
```

Comment:
Shouldn't `soaEditApi` default to `soaEdit`?

Reply:
PowerDNS does this internally